pysolarmanv5/pysolarmanv5.py

```diff
@@ -269,7 +272,7 @@ def _v5_frame_decoder(self, v5_frame):
             raise V5FrameError("V5 frame contains invalid sequence number")
         if v5_frame[7:11] != self.v5_loggerserial:
             raise V5FrameError("V5 frame contains incorrect data logger serial number")
-        if v5_frame[3] != self.v5_magic and v5_frame[4] != CONTROL.REQUEST - 0x30:
+        if v5_frame[3] != self.v5_magic or v5_frame[4] != CONTROL.REQUEST - 0x30:
```

Comment:
For clarity, it would be easier if we stuck with either the existing definition of a two byte control code, or else redefine that into two, new, single byte fields. We refer to `0x4510` in some places, and now we are refering to just `0x45`. Same with `0x3000` vs `0x30`.

Reply:
You reviewed outdated code.

Reply:
But yes, I'm all for separating those bytes cause handling it that way (using int) is way easier.